src/pages/PokemonDetailsPage/PokemonDetailsPage.css

```diff
@@ -21,6 +22,7 @@
   outline: none;
   font-size: 2rem;
   width: fit-content;
+  height: 1.5em;
```

Comment:
I noticed pokemon with longer names made the pokemon title wrap around. 

These changes make it so when that happens the copy to clipboard button stays the same size and is centred.